etna/analysis/eda/plots.py

```diff
@@ -341,7 +341,7 @@ def plot_holidays(
 
     for i, segment in enumerate(segments):
         segment_df = df.loc[start:end, pd.IndexSlice[segment, "target"]]  # type: ignore
-        segment_df = segment_df[segment_df.first_valid_index() : segment_df.last_valid_index()]
+        segment_df = segment_df.loc[segment_df.first_valid_index() : segment_df.last_valid_index()]
```

Comment:
How can we find all such places? If it possible to handle the search via type checking or linters? Looks like we miss many of them

Reply:
I don't know how to find them by using type checking of linters. I searched for `df[` in a repository.